spring-cloud-app-broker-deployer-cloudfoundry/src/main/java/org/springframework/cloud/appbroker/deployer/cloudfoundry/CloudFoundryDeploymentProperties.java

```diff
@@ -55,6 +55,8 @@
 
 	static final String JAVA_OPTS_PROPERTY_KEY = "javaOpts";
 
+	public static final long DEFAULT_API_POLLING_TIMEOUT_SECONDS = Duration.ofMinutes(5).getSeconds(); // Default in cf-java-client when no completionTimeout is specified in CSI/USI/DSI
```

Comment:
Please, remove the comment, the name seems explicit enough.

Reply:
thanks, this is fixed with  bd25634